File: proselint/checks/cliches/hell.py
"""
Too much yelling.

---
layout:     post
source:     ???
source_url: ???
title:      yelling
date:       2014-06-10 12:31:19
categories: writing
---

Never use the phrase 'all hell broke loose'.

"""

from proselint.registry.checks import Check, types

check = Check(
    check_type=types.ExistenceSimple(pattern="all hell broke loose"),
    path="cliches.hell",
    message="Never use the phrase 'all hell broke loose'.",
)

__register__ = (check,)

File: proselint-rs/crates/proselint-checks/src/cliches/hell.rs
use proselint_registry::checks::{Check, types::*, CheckFlags};

pub const EXAMPLES_PASS: &[&str] = &["Smoke phrase with nothing flagged."];
pub const EXAMPLES_FAIL: &[&str] = &["I was at xyz and then all hell broke loose."];

const CHECK: Check = Check {
	check_type: &ExistenceSimple {
		pattern: "all hell broke loose",
		exceptions: &[],
	},
	path: "cliches.hell",
	msg: "Never use the words 'all hell broke loose'.",
	flags: CheckFlags { limit_results: 3, ppm_threshold: 0 },
	..Check::default()
};

pub const REGISTER: &[Check] = &[CHECK];
